misc/python/materialize/cli/mzcompose.py

```diff
@@ -27,6 +27,21 @@
 announce = ui.speaker("==>")
 say = ui.speaker("")
 
+MIN_DOCKER_VERSION = (1, 27, 0)
+
+
+def assert_docker_compose_version() -> None:
+    """Check the version of docker-compose installed.
+
+    Raises `MzRuntimeError` if the version is not recent enough.
+    """
+    cmd = ["docker-compose", "version", "--short"]
+    output = subprocess.check_output(cmd).decode("utf-8").strip()
```

Comment:
We have a nice wrapper for this: ``` output = spawn.capture(["docker-compose", "version", "--short"], unicode=True)```

Reply:
(You'll need to `from materialize import spawn`.)

Reply:
Done!